java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala

```diff
@@ -33,6 +33,8 @@ class TestOpenmldbCatalogService extends FunSuite {
       println(s"Database $i: ${databases(i)}")
     }
 
+    val internalTableNames = List("JOB_INFO", "PRE_AGG_META_INFO")
+    assert(internalTableNames.contains(tables(i)))
```

Comment:
You need to get `tables(i)` in the following `for` loop.

Reply:
I am not sure, In scala how to do that.

Reply:
I am not sure, In scala how to do that.

Reply:
Jost move these added code in line 40.